src/helpers.ts

```diff
@@ -123,9 +156,9 @@ export async function queryWebsoc(params) {
     }
 }
 
-const gradesCache = {};
+const gradesCache: {[key: string]: any} = {};// TODO replace this any
```

Comment:
todo in this pr or later?

Reply:
added interface to replace the any: https://github.com/icssc/AntAlmanac/pull/444/commits/76ad65422efe11142cd009588bc7d5175d892f1f
```
interface Grades {
    average_gpa: number;
    sum_grade_a_count: number;
    sum_grade_b_count: number;
    sum_grade_c_count: number;
    sum_grade_d_count: number;
    sum_grade_f_count: number;
    sum_grade_np_count: number;
    sum_grade_p_count: number;
}

const gradesCache: {[key: string]: Grades} = {};
```